MSAL/src/public/MSALWebviewParameters.h

```diff
@@ -94,6 +94,10 @@ NS_ASSUME_NONNULL_BEGIN
 */
 - (nonnull instancetype)initWithAuthPresentationViewController:(MSALViewController *)parentViewController;
 
+
+@property (nonatomic, readonly) WKWebViewConfiguration *defaultWKWebviewConfiguration;
```

Comment:
Should this be a class property? Also, let's add documentation similar to ADAL.

Reply:
good point. Thank you.